.travis.yml

```diff
@@ -35,17 +35,13 @@ install:
     - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
         conda install gxx_linux-64 libgcc -c conda-forge;
       fi
-    # Test dependencies
-    - conda install pytest jupyter_kernel_test -c conda-forge;
```

Comment:
You might want to keep this. You will need it later.

Reply:
Hm, really? haha
Ok. Will add it back in.

Reply:
Yes, when adding Python tests like in xeus-python and xeus-cling ;) I will help you doing it